### DISCUSSION TITLE
How to stop the rate limit logging

### DISCUSSION THREAD

**Discussion by ghost** [disgoorg/disgo#405] [Help] [answered]
So i am trying to write a application which sends webhook to a channel to report about some stats and it sometimes has to do it fast so the rate limiter tries to rate limit it but it then prints a warn message with the webhook in logs which i dont want cuz sometimes i need to share logs publically and i like the webhook warning that prints the webhook is anoying

here is my basic code
```go
webhook_logger := logrus.New()
webhook_logger.SetLevel(logrus.FatalLevel)
client, err := webhook.NewWithURL(catbox.G_config.WebhookURL, webhook.WithLogger(webhook_logger))
if err != nil {
	app.G_logger.Errorln(err)
	os.Exit(1)
}
```

so how do i stop it from printing the warn message with my webhook url

**Comment by topi314** [comment, marked as answer]
you can pass a logger with the level set to `ERROR` to the ratelimiter used by the webhook
```go
webhook.NewWithURL(url,
	webhook.WithRestClientConfigOpts(
		rest.WithRateLimiterConfigOpts(
			rest.WithRateLimiterLogger(logger),
		),
	),
)
```
